osu.Game/Localisation/MenuTipStrings.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class MenuTipStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.MenuTip";
+
+        /// <summary>
+        /// "Press Ctrl-T anywhere in the game to toggle the toolbar!"
+        /// </summary>
+        public static LocalisableString ToggleToolbarShortcut => new TranslatableString(getKey(@"toggle_toolbar_shortcut"), @"Press Ctrl-T anywhere in the game to toggle the toolbar!");
+
+        /// <summary>
+        /// "Press Ctrl-O anywhere in the game to access options!"
```

Comment:
Was renamed in lazer.
```suggestion
        /// "Press Ctrl-O anywhere in the game to access settings!"
```